packages/termux-tools/termux-change-repo

```diff
@@ -43,7 +43,16 @@ select_repository() {
         SCIENCE="https://dl.kcubeterm.me/science-packages-24"
         UNSTABLE="https://dl.kcubeterm.me/unstable-packages"
         X11="https://dl.kcubeterm.me/x11-packages"
-
+        
+	elif [ "$1" == "Mirrors by Kcubeterm*" ]; then
```

Comment:
Maybe change to for example "GH mirrors by Kcubeterm"?

Reply:
Yup, This is awesome.